include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -322,6 +371,31 @@ namespace parallel
                            const std::vector<dealii::CellData<dim>> &cells,
                            const SubCellData &subcelldata) override;
 
+      /**
+       * Implementation of the same function as in the base class.
+       *
+       * @param other_tria The triangulation to be copied. It can be a serial
+       *        Triangulation or a parallel::distributed::Triangulation. Both
+       *        can have been refined already.
+       *
+       * @note Please note that this function requires that a
```

Comment:
I think we overuse "Please note", especially if it's preceded by `@note` :-)
```suggestion
       * @note This function requires that a
```